app-static/app-static.pro

```diff
@@ -77,10 +77,7 @@ HEADERS += \
 #
 # Add search paths below /usr/local for Mac OSX
 #
-macx {
-  LIBS += -L/usr/local/lib
```

Comment:
If the headers are in /usr/local/include, would it make sense to have the libs in /usr/local/lib?